README.md

```diff
@@ -52,39 +52,56 @@ Nightfall provides pre-built detector types, covering data types ranging from PI
 
 #### Sample Code
 
+To run this sample script you must compile it as TypesScript. Save it as a `.tsc` file and run `tsc <yourfilename>.ts -lib ES2015,DOM `
+
+You can this run the resulting JavaScript file:
+
+`NIGHTFALL_API_KEY=<YourApiKey> node nodesdksample.js`
+
 ```typescript
 // By default, the client reads your API key from the environment variable NIGHTFALL_API_KEY
+import { Nightfall } from "nightfall-js";
+import { Detector } from "nightfall-js/dist/types"
+
 const nfClient = new Nightfall();
 
-const response = await nfClient.scanText(['My credit card number is 4242-4242-4242-4242'], {
-  detectionRules: [
-    {
-      name: 'Secrets Scanner',
-      logicalOp: 'ANY',
-      detectors: [
-        {
-          minNumFindings: 1,
-          minConfidence: Detector.Confidence.Possible,
-          displayName: 'Credit Card Number',
-          detectorType: Detector.Type.Nightfall,
-          nightfallDetector: 'CREDIT_CARD_NUMBER',
-        },
-      ],
-    },
-  ],
-});
-
-if (response.isError) {
-  console.log(response.getError());
-} else {
-  response.data.findings.forEach((finding) => {
-    if (finding.length > 0) {
-      finding.forEach((result) => {
-        console.log(`Finding: ${result.finding}, Confidence: ${result.confidence}`);
-      });
-    }
+var scanText = async function () {
+  return await nfClient.scanText(['The customer social security number is 111-22-3333'], {
+    detectionRules: [
+      {
+        name: 'Secrets Scanner',
+        logicalOp: 'ANY',
+        detectors: [
+          {
+            minNumFindings: 1,
+            minConfidence: Detector.Confidence.Possible, //'POSSIBLE', //
+            displayName: 'SSN Detector',
+            detectorType: Detector.Type.Nightfall, //'NIGHTFALL_DETECTOR', //
+            nightfallDetector: 'US_SOCIAL_SECURITY_NUMBER',
+          },
+        ],
+      },
+    ],
   });
+};
+
+async function main() {
+  const response = await scanText();
+
+  if (response.isError) {
+    console.log(response.getError());
+  } else if (response.data) {
+    response.data.findings.forEach((finding) => {
+      if (finding.length > 0) {
+        finding.forEach((result) => {
+          console.log(`Finding: ${result.finding}, Confidence: ${result.confidence}`);
+        });
+      }
+    });
+  }
 }
+
+main();
```

Comment:
Hmm.. I'm just a little hesitant to approve this because it is fairly uncommon to break down the call like this. Most people will call our API and loop through the responses in the same function. Why not move the API call into `main()` and remove `scanText()`?

Reply:
I don't think it's necessarily the worst decomposition in the world, but we can simply to make it consistent with file scan example